### ISSUE TITLE
Any pre-training in Kinetics-400 to get 45.6% top-1 in Sth-Sth V1?

### ISSUE THREAD

**Issue by Andy1621** [mit-han-lab/temporal-shift-module#70]
Hello, thanks for your great job again!!!
I have noticed that in your paper firtly published in arxiv, the accuracy are lower than your results in ICCV. And I find that in this repo, the TSM-Sth-Sth-v1 is tued from the TSM-Kinetics400. Could you please tell me is there any pre-training in Kinetics-400 to get 45.6% top-1 in Sth-Sth V1? 
BTW, the pretrained model of Sth-Sth V1 only gets ~20% top1, and I trained it for another epoches to get 45.5% top1......
#65

**Comment by tonylins** [comment]
Hi, for something-something V1&V2 dataset, we did not use Kinetics pre-training in the v2 arXiv paper (which gets higher accuracy). Kinetics pre-training is worse than ImageNet pre-training on these datasets.

For pre-trained model performance, I think the problem might be the pre-processing.
